cmd/codegen/app/generate_docs.go

```diff
@@ -74,17 +76,24 @@ func run(o *GenerateDocsOptions) error {
 	}
 	util.AppLogger().Infof("generating docs to %s\n", o.OutputBase)
 
-	err = generator.InstallGenAPIDocs(o.GeneratorVersion)
+	gopath := util.GoPath()
```

Comment:
Could this handling of the isolated GOPATH creation not be handled in something like `func GoPath(isolated bool)`?

Reply:
yes, it could.

Reply:
but not worth doing?